docs/modules/ROOT/partials/getting-started/con-registry-client.adoc

```diff
@@ -3,6 +3,18 @@
 [id="registry-client-intro"]
 = {registry} Java client overview
 
-You can manage artifacts stored in {registry} using a Java client application. You can create, read, update, or delete artifacts stored in the registry using the {registry} Java client classes...  
+You can manage artifacts stored in {registry} using a Java client application. You can create, read, update, or delete artifacts stored in the registry using the {registry} Java client classes.
 
-More TBD
+You can access the {registry} Java client by adding the proper dependency to your project, see xref:writing-registry-client[].
+
+The {registry} client is autocloseable and is implemented using Retrofit and OkHttp. This gives the user the possibility of customize its usage like, for example, adding custom headers or enabling TLS auth support.
```

Comment:
Suggest: 
...implemented using the OkHttp and Retrofit HTTP clients

Reply:
I don't think this applies since we're not using just the client, the intention here is to mention them as the base libraries for the client.

Reply:
OK cool, maybe say the "...OkHttp and Retrofit base libraries" or leave as is if you think that's confusing?